tasks/managed/push-rpm-to-koji/push-rpm-to-koji.yaml

```diff
@@ -56,8 +57,13 @@ spec:
 
         PRINCIPAL=$(jq -r '.pushOptions.pushKeytab.principal' "${DATA_FILE}")
         KEYTAB_FILE=$(jq -r '.pushOptions.pushKeytab.name' "${DATA_FILE}")
-        KOJI_CMD=$(jq -r '.pushOptions.koji_cmd' "${DATA_FILE}")
+        KOJI_PROFILE=$(jq -r '.pushOptions.koji_profile' "${DATA_FILE}")
         KOJI_TAGS=$(jq -r '.pushOptions.koji_tags  | join(" ")' "${DATA_FILE}")
+        KOJI_IMPORT_DRAFT=$(jq -r '.pushOptions.koji_import_draft' "${DATA_FILE}")
```

Comment:
All of these are needed, but if they are not set, it will not end well. For the string ones, the result of the jq command would be the string "null". For the tags, you would get: `jq: error (at <stdin>:1): Cannot iterate over null (null)`

It would be good to handle these - for the strings, set some default values via "//". Or if we really need those, there should be a check and the script should `exit 1` or something. For the tags, maybe `jq -r '.pushOptions.koji_tags  // [] | join(" ")'` would be good enough.

Reply:
I thought these are required by the schema, which is also used to verify the types. Or do we need to include the input data validation in the pipeline? Is there an example how it should be handled?

Reply:
We have a schema file, but everything is optional, unless you define a "system" in the schema and then use it in the pipeline like here: https://github.com/konflux-ci/release-service-catalog/blob/development/pipelines/managed/push-artifacts-to-cdn/push-artifacts-to-cdn.yaml#L212

A lot of things are also validated in the konflux-release-data repo, so likely once products start using it, some checks will be added. But I think it would be good to still have at least something even in the task directly - for example some default values where it makes sense. But it's just a suggestion. If you think you should be able to rely on the fields being correct, it's fine with me.

Reply:
I've added the fallback (`[]`) and minimal verification to ensure the data are provided (using `jq --exit-status`).

Reply:
ok, thanks